adresy/draft/příklady/3.csv-metadata.json

```diff
@@ -16,7 +16,11 @@
                 "name": "typ_čísla_domovního",
                 "titles": "typ_čísla_domovního",
                 "dc:description": "Typ čísla domovního, může nabývat hodnot: č.p.- číslo popisné, č.ev.- číslo evidenční.",
-				"datatype": "string"
+				"datatype": "string",
+        "enum": [
+						"č.p.",
+						"č.ev."
+				]

```

Comment:
Nevypadá to, že by v [CSV on the Web](https://www.w3.org/TR/2015/REC-tabular-metadata-20151217/) byla podpora pro enumy...

Reply:
to jsem nevěděl, smažu (jel jsem podle json schématu)